src/main/app.ts

```diff
@@ -125,6 +126,9 @@ app.use((err, req, res, next) => {
     res.render(new ForbiddenError().associatedView)
   } else {
     const view = FeatureToggles.isEnabled('returnErrorToUser') ? 'error_dev' : 'error'
+    if (err.name === 'Template render error') {
+      trackCustomEvent('CMC Dashboard Failure', { error: err })
```

Comment:
I believe andy is changing this in his PR so might want to check for conflicts: https://github.com/hmcts/cmc-citizen-frontend/pull/2014

Reply:
I have @andrewwa-kainos 's commit in my PR, it will not show here once other PR is merged to master.